### PR TITLE
[Sonos] Updated and added complete german translations

### PR DESCRIPTION
**Pull request by cweitkamp**
Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/i18n/sonos_de.properties [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+channel-type.sonos.currentalbumart.label = Coverbild
+channel-type.sonos.currentalbumart.description = Zeigt das Coverbild des aktuellen St�cks an.
+channel-type.sonos.currentalbumarturl.label = Coverbild URL
+channel-type.sonos.currentalbumarturl.description = Zeigt die URL zum Coverbild des aktuellen St�cks an.
+channel-type.sonos.currentartist.label = K�nstler
+channel-type.sonos.currentartist.description = Zeigt den K�nstler des aktuellen St�cks an.
+channel-type.sonos.currenttitle.label = Titel
+channel-type.sonos.currenttitle.description = Zeigt den Titel des aktuellen St�cks an.
+channel-type.sonos.currenttrack.label = Track
+channel-type.sonos.currenttrack.description = Zeigt den Track des aktuellen St�cks an.
```
Comment (htreu) [inline extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/i18n/sonos_de.properties]: Should this be more specific be "Track-Nummer"?

File: extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/i18n/sonos_de.properties [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+channel-type.sonos.currenttrackuri.label = Track URI
+channel-type.sonos.currenttrackuri.description = Zeigt die URI des aktuellen Tracks an.
+channel-type.sonos.currenttransporturi.label = AV Transport URI
+channel-type.sonos.currenttransporturi.description = Zeigt die URI des aktuellen AV Transportes an.
+channel-type.sonos.favorite.label = Favorit
+channel-type.sonos.favorite.description = Erm�glicht das Abspielen des Favoriten. Einstellung von Favoriten m�ssen in der Sonos Controller App erfolgen.
```
Comment (htreu) [inline extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/i18n/sonos_de.properties]: Is this only one favorite? Or should it be "... eines Favoriten."?

File: extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/i18n/sonos_de.properties [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+channel-type.sonos.shuffle.description = Erm�glicht das Abspielen der aktuellen Wiedergabeliste in zuf�lliger Reihenfolge.
+channel-type.sonos.sleeptimer.label = Sleep Timer
+channel-type.sonos.sleeptimer.description = (Ein-)Schlaffunktion (in Sekunden).
+channel-type.sonos.snooze.label = Snooze
+channel-type.sonos.snooze.description = Erm�glicht die Snooze-Funktion f�r den aktuellen Alarm (in Minuten).
+channel-type.sonos.standalone.label = Stand-Alone
+channel-type.sonos.standalone.description = Entfernt den aktuellen Sonos Speaker aus der aktuellen Gruppe und stellt ihn als Stand-Alone Speaker bereit.
+channel-type.sonos.state.label = Status
+channel-type.sonos.state.description = Zeigt den Status des Sonos Speakers an (z.B. PLAYING, STOPPED, ...) 
+channel-type.sonos.stop.label = Stop lecture
```
Comment (htreu) [inline extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/i18n/sonos_de.properties]: This is still the french translation

File: extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/i18n/sonos_de.properties (hunk 2 above)
Comment (cweitkamp) [reply extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/i18n/sonos_de.properties]: Yes, it is only one favorite. I will change ist to '... des eingestellten Favoriten.'.

File: extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/i18n/sonos_de.properties (hunk 1 above)
Comment (cweitkamp) [reply extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/i18n/sonos_de.properties]: I agree. But it is a String item. We should translate it more closely to the english description. It should be `Zeigt den Namen des aktuellen Tracks oder des aktuellen Radiosenders an.`.

**Pull request merged**